cmake/ConkyBuildOptions.cmake

```diff
@@ -156,6 +156,7 @@ option(BUILD_WAYLAND "Build Wayland support" false)
 option(BUILD_X11 "Build X11 support" true)
 if(BUILD_X11)
   option(OWN_WINDOW "Enable own_window support" true)
+  option(MOUSE_EVENTS "Enable mouse event support" true)
```

Comment:
In keeping with the format of the options, change this from `MOUSE_EVENTS` to `BUILD_MOUSE_EVENTS`.